src/libraries/System.Net.Connections/tests/System.Net.Connections.Tests/System.Net.Connections.Tests.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <!-- Test on same targets as System.Net.Socket.Tests -->
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
```

Comment:
I ended up having [weird ThreadPool-related PNSE-s](https://dev.azure.com/dnceng/public/_build/results?buildId=766288&view=ms.vss-test-web.build-test-results-tab&runId=23815156&paneView=debug&resultId=124647) on some platforms. Decided to run the tests on same platforms we run socket tests on. Hope this is fine.

Reply:
Seems fine to me